tox.ini

```diff
@@ -24,6 +24,8 @@ format = pylint
 [testenv:syntax]
 deps = 
     flake8
+    # NOTE(retr0h): We specifically pin to a known version so formatting
+    # is consistent across developer/PR.
     yapf==0.10.0
```

Comment:
> That's only one use case. For instance my editor needs know where yapf is, because it reformats my the files every time I save them.

But that happens to be your use case.  Most ppl don't have this wired into their editor.  To be honest, I do the same thing with my editor, and since I am very savy with the project, I have the proper yapf installed on my system.  However, I am trying to keep things simple for the community at large.